src/admin/components/SearchResult.js

```diff
@@ -26,14 +27,15 @@ const ResultLink = ({ children, end, middle, start, ...props }) =>
       'button-middle': middle,
       'button-start': start,
     })} rel="noopener noreferrer" target="_blank" {...props} >{children}</a>;
+
```

Comment:
I found it really hard to read with everything crammed up against each other. I know the lines are related though.

If it's convention though I'll accept I just need to acclimatise though.